docs/mine-hnt/validators/wallet.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: validator-wallet
+title: Configure a Wallet
+hide_title: true
+sidebar_label: Configure a Wallet
+slug: /mine-hnt/validators/validator-wallet
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Configure a wallet
+
+You’ll need an owner wallet to:
+* Stake your Testnet Network Tokens (TNT)
+* Receive validator node consensus group rewards
+
+Note: you can have a single wallet for multiple validator nodes. However multiple wallets for a single validator node is not supported.
+
+For instructions to build the wallet go [here](https://github.com/helium/helium-wallet-rs).
+
+At any time use ```-h``` or ```--help``` to get more help for a command.
+
+## Wallet commands
+
+### Create a wallet
+
+```./bin/helium-wallet create basic --network testnet``` 
+
+### View wallet balance and public key
+```./bin/helium-wallet info``` view public key and TNT balance. You’ll need this public key address to obtain TNT needed for staking. 
+
+To obtain TNT tokens visit [Obtain Testnet Network Tokens (TNT)](/mine-hnt/validators/validator-tnt).
+
+:::warning
+
+ Always make sure to inspect the transaction _before_ adding the **--committ**  option to sign and submit it to the api. There is no undo.
+
+:::
+
+### Stake tokens
```

Comment:
I think this section should be expanded a bit to explain the staking mechanics. E.g 

* 10000 TNT to stake
* TNT is stored in wallet A, and is staked using a transaction but not actually sent to the validator node itself. 
* Etc